app/views/LocationShare/LiveLocationPreviewModal.tsx

```
@@ -0,0 +1,522 @@
+		return;
+	}
+
+	const params = globalTrackerParams;
+	try {
+		const state = globalTracker.getCurrentState();
+		globalTracker.stopTracking();
+
+		if (params?.liveLocationId) {
+			try {
+				markLiveLocationAsEnded(params.liveLocationId);
+			} catch {}
```

ESLint

[error] Empty block statement.

(no-empty)